spec/game_spec.rb

```diff
@@ -0,0 +1,336 @@
+require './game.rb'
+require './player.rb'
+
+RSpec.describe Game do
+
+    before Game do 
+        @player1 = Player.new('John','X')
+        @player2 = Player.new('Doe','O')
+        @game = Game.new(@player1,@player2) 
+    end
+           
+
+    describe '#set_board_value' do 
+    
+
+        it 'Adds X to [0,0] position of the board' do
```

Comment:
Great work testing all possible inputs. We can DRY this code out by useing RSpec's [Shared Examples](https://medium.freecodecamp.org/how-to-dry-out-your-rspec-tests-using-shared-examples-d5cc5d33fd76).